blocks/text.js

```diff
@@ -825,18 +830,24 @@ Blockly.Constants.Text.TEXT_CHARAT_MUTATOR_MIXIN = {
         this.appendDummyInput('ORDINAL')
             .appendField(Blockly.Msg.ORDINAL_NUMBER_SUFFIX);
       }
-    } else {
-      this.appendDummyInput('AT');
     }
     if (Blockly.Msg.TEXT_CHARAT_TAIL) {
       this.removeInput('TAIL', true);
       this.appendDummyInput('TAIL')
           .appendField(Blockly.Msg.TEXT_CHARAT_TAIL);
     }
-    var menu = new Blockly.FieldDropdown(this.WHERE_OPTIONS, function(value) {
+
+    this.isAt_ = isAt;
+  }
+};
+
+// Does the initial mutator update of text_charAt and adds the tooltip
+Blockly.Constants.Text.TEXT_CHARAT_EXTENSION = function() {
```

Comment:
The contents of this function should be unindented by two spaces.

Reply:
done.